client-ng1/src/app/js/admin/cfp/cfp.controller.js

```diff
@@ -22,8 +22,42 @@
           ctrl.proposals = response.data;
         }
       );
+      $http.get('app/cfp/proposal/votes').then(
+          function(response){
+            ctrl.votesMappedByProposalId = {};
+            angular.forEach(response.data, function(data) {
+              ctrl.votesMappedByProposalId[data.proposal.id] = data.voteValue;
```

Comment:
Because I had to be quick and go to bed ;-)
I'll have to review that, I think I have way too much data serialized here (like the full proposal and the full staff entities).